src/complex/DataStructure/DataStore.hpp

```diff
@@ -364,17 +364,27 @@ class DataStore : public AbstractDataStore<T>
     return err;
   }
 
-  static std::unique_ptr<DataStore> ReadHdf5(const H5::DatasetReader& datasetReader)
+  static std::unique_ptr<DataStore> ReadHdf5(const H5::DatasetReader& datasetReader, bool isNeighborList = false)
```

Comment:
I think reading neighbor list data from HDF5 should be its own separate function since neighbor list don't even use `DataStore` and end up discarding them anyway creating unnecessary copies.

Reply:
fixed by aeae3636dfd6ed58fcffe6dea0c08c6668b5120c